test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -184,7 +184,9 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 				}
 			}
 		}
-		framework.ExpectEqual(eventFound, true)
+		if !eventFound {
+			framework.Failf("Appropriate event was not generated, there is no object invloved which have both kind == Pod and e.Reason == NotTriggerScaleUp ")
```

Comment:
```suggestion
			framework.Failf("Expected event with kind 'Pod' and reason 'NotTriggerScaleUp' not found.")
```

Reply:
In previous PR's there was an error that I got when formatting directive was not used in Failf function. In this case also we have not used any, is it mandatory to use them or not in failf func.

Reply:
It would be nicer to use a `framework.Fail` when the first parameter is not meant to be a format string, but such a function doesn't exist. It's not required for it to be a format string.

What is required is to match format specifiers (like `%s`) with the corresponding parameters.